sdk/data/azappconfig/ci.yml

```diff
@@ -27,3 +27,4 @@ extends:
       ServiceDirectory: 'data/azappconfig'
       RunLiveTests: true
       UsePipelineProxy: false
+      UseFederatedAuth: true
```

Comment:
I assume this is for testing?

Reply:
It is. Comment added, will be removed before merge.